app/components/record_details/view.rb

```diff
@@ -103,7 +105,10 @@ def render_text_with_hint(date)
     end
 
     def trainee_start_date
-      trainee.commencement_date.present? ? date_for_summary_view(trainee.commencement_date) : nil
+      return t(".course_not_started") if trainee.starts_course_in_the_future?
```

Comment:
Oh no.. and i think @mooktakim has a ticket for displaying the trainee start date! 😓

Reply:
Yep, this method is already different on main based on what @archferns merged recently. 
Here's my card:
https://trello.com/c/3DnYjTL4/3161-trainee-start-date-row-record-view